alibi/utils/distance.py

```diff
@@ -122,8 +122,8 @@ def abdm(X: np.ndarray,
     # combine dict for categorical with binned features
     cat_vars_combined = {**cat_vars, **cat_vars_bin}
 
-    d_pair = {}  # type: Dict
-    X_cat_eq = {}  # type: Dict
+    d_pair: dict = {}
```

Comment:
Just noting that elsewhere we've done `dict` -> `Dict` but here it is `Dict -> dict`